clients/tfchain-client-go/node.go

```diff
@@ -688,6 +688,32 @@ func (s *Substrate) SetNodePowerState(identity Identity, up bool) (hash types.Ha
 	return callResponse.Hash, nil
 }
 
+// ChangePowerTarget updates the power target
+func (s *Substrate) ChangePowerTarget(identity Identity, nodeID uint32, up bool) (hash types.Hash, err error) {
```

Comment:
Why call it Change when every other method in this client uses `Set`

Reply:
I know the chain is using the `change` name but it's wrong. The Client doesn't have to use the same conviention